authmodules/TYPO3/privacyidea/ext_conf_template.txt

```diff
@@ -11,7 +11,7 @@ privacyIDEAURL = https://localhost
 privacyIDEACertCheck = 0
 
 # cat=Server/Optional; type=string; label=privacyIDEA realm
-privacyIDEARealm = ""
+privacyIDEARealm = defrealm
```

Comment:
OK I have tested it and if I leave it empty then the default realm is used. As a result this line should be:
`privacyIDEARealm = `